lib/gulp/css.js

```diff
@@ -16,7 +17,23 @@ function css(cb) {
   cb();
 }
 
-function cssCompile({src, dest, minified = true, browserSync = false}) {
+async function getPostCSSConfiguration(minified) {
+  try {
+    // Get PostCSS config from user
+    return await loadPostCSSConfig();
+  } catch (error) {
+    // Catch no config found from postcss-load-config in order to set default
+    // values for our PostCSS instance.
+    if (minified) {
+      return {plugins: [rfs(), autoprefixer(), cssnano()]};
+    }
+
+    return {plugins: [rfs(), autoprefixer()]};
+  }
+}
+
+async function cssCompile({src, dest, minified = true, browserSync = false}) {
+  const {plugins, options = {}} = await getPostCSSConfiguration(minified);
```

Comment:
I found the little culprit..

The function that loads the PostCSS config wasn't called in a blocking way so it continued and tried to initiate gulp-postcss before actually getting the PostCSS config therefore gulp-postcss threw an error that I didin't handle.

Now that its blocking there is always a config available. Either the config from the user or the default one we provide.